README.md

```diff
@@ -145,6 +145,10 @@ Check out [node-pg-pool](https://github.com/brianc/node-pg-pool) and [node postg
       <td>Boolean</td>
       <td>Whether to try SSL/TLS to connect to server</td>
     </tr>
+    <tr>
+      <td>defaultSort</td>
```

Comment:
I am not good at naming :grimacing: would like to hear from the team @strongloop/sq-lb-apex

Reply:
Maybe `defaultIdSort` since it only applies to the `id` field? But I'm not good at naming either 😅